docs/source/topics/processes/concepts.rst

```diff
@@ -142,6 +142,7 @@ When a process is 'submitted', an instance of the ``Process`` is created, along
 This is called a 'process checkpoint', more information on which :ref:`will follow later<topics:processes:concepts:checkpoints>`.
 Subsequently, the process instance is shut down and a 'continuation task' is sent to the process queue of RabbitMQ.
 This task is simply a small message that just contains an identifier for the process.
+In order to reconstruct the process from 'process checkpoint', the process needs to be importable in the daemon environment by a) giving it an associated entry point or b) including its module path in the ``PYTHONPATH`` that the daemon workers will have.
```

Comment:
```suggestion
In order to reconstruct the process from a `checkpoint`, the process needs to be importable in the daemon environment by a) giving it an :ref:`associated entry point<how-to:plugin-codes:entry-points>` or b) :ref:`including its module path<how-to:faq:process-not-importable-daemon>` in the ``PYTHONPATH`` that the daemon workers will have.
```
You will have to add the line
```
.. _how-to:faq:process-not-importable-daemon:`
```
to the file `docs/source/howto/faq.rst` on line 65 for the second link